SteamKit2/Tests/NetworkListenerFacts.cs

```diff
@@ -0,0 +1,42 @@
+﻿using Xunit;
+using SteamKit2;
+using SteamKit2.Internal;
+
+namespace Tests
+{
+    class TestNetworkListener : IDebugNetworkListener
+    {
+        public EMsg NextExpectedSentMessage;
```

Comment:
This is backwards from how I've usually seen tests like this - store the last one in the stub, assert from the test itself. Here you have no control from the test itself about where this runs.